content/blog/2019-10-21-rmangal.md

```diff
@@ -0,0 +1,154 @@
+did so because we needed feedback, we needed qualified people to critically
+assess whether our two main objectifs were achieved. Given the strong expertise
+of rOpenSci in software review, and given that our package was in-scope,
+submitting rmangal to rOpenSci was an obvious choice.
+
+We [had very valuable
+feedback](https://github.com/ropensci/software-review/issues/332) from [Anna
+Willoughby][arw36] and [Thomas Lin Pedersen][thomasp85]. They carefully assessed
+our work and pointed out areas where improvement was required. One good example
+of how their review made our package better concerns the dependencies. We
+originally listed [sf][sf] in `Imports` as we used it to filter networks based
+on geographic coordinates. But the reviewers pointed out that this was not an
+essential part of the package and that [sf][sf] has several dependencies. This  made us realize that for one extra feature, we were substantially
+increasing the number of indirect dependencies. Following the reviewers'
+suggestions, we moved [sf][sf] to `Suggests` and added a message to warn users
+that to use the [spatial filtering
+feature](https://ropensci.github.io/rmangal/articles/rmangal.html#geolocalize-mangal-networks-with-sf)
+requires sf to be installed. Similarly, based on another good comment, we
+added a function to convert Mangal networks into [tidygraph][tidygraph] objects
+and we documented how to plot Mangal networks with [ggraph][ggraph] (and so we
+added those packages in `Suggests`). Such improvements were very helpful to
+properly connect rmangal to the existing R packages. The plethora of R packages
+is one of its major strength, and connecting a package properly to other makes
+the entire ecosystem even stronger.
+
+<!-- Future -->
+We are now looking for user feedback, not only for rmangal but also for the web
+API and the website. We welcome suggestions and contributions, especially for
+the documentation. In the future, we envision that rmangal will integrate
+functions to format ecological networks for ecologists willing to add their
+datasets to Mangal. This will likely be the next major release of rmangal.
+
+
+
+### Acknowledgments
+
+We are thankful to all [contributors to
+rmangal](https://ropensci.github.io/rmangal/authors.html) and to all ecologists
+that have spent countless hours in collecting data. We would like to thank [Anna
+Willoughby][arw36] and [Thomas Lin Pedersen][thomasp85] for thorough reviews as
+well as [Noam Ross](https://github.com/noamross) for handling the review
+process.
+
+[^original_ref]: Poisot, T. et al. mangal - making ecological network analysis simple. Ecography 39, 384–390 (2016).
+
+
+[arw36]: https://github.com/arw36
+[thomasp85]: https://github.com/thomasp85
+[igraph]: https://igraph.org/r/
+[tidygraph]: https://tidygraph.data-imaginist.com/
+[ggraph]:https://ggraph.data-imaginist.com/
+[rmangal]: https://ropensci.github.io/rmangal/
```

Comment:
Consider linking to https://docs.ropensci.org/rmangal/ (automatically built and hosted documentation) and adding that url to https://github.com/ropensci/rmangal. This is 

About: https://ropensci.org/technotes/2019/06/07/ropensci-docs/

Reply:
https://docs.ropensci.org/rmangal/ added on the repo! Do you want to change it in the post?

Reply:
Can you please make the changes in the post yourself, and then once you've finished I can make minor changes that arise later if needed

Reply:
Will do :+1: !